Components/FixedImagePyramids/OpenCLFixedGenericPyramid/elxOpenCLFixedGenericPyramid.hxx

```diff
@@ -196,8 +196,12 @@ OpenCLFixedGenericPyramid<TElastix>::GenerateData()
 
   if (computedUsingOpenCL)
   {
-    // Graft output
-    this->GraftOutput(this->m_GPUPyramid->GetOutput());
+    // Graft outputs
+    const auto numberOfLevels = this->GetNumberOfLevels();
+    for (unsigned int i = 0; i < numberOfLevels; i++)
```

Comment:
FYI: No show-stopper, but in general, in C++ code, `++i` is preferred to `i++`, see pull request #467. Just a minor style issue in this particular case. No action needed.